libs/sdk-backend-tiger/tests/integrated/insights.test.ts

```diff
@@ -10,14 +10,14 @@ beforeAll(async () => {
 describe("tiger insights", () => {
     it("should load insights", async () => {
         const result = await backend.workspace(testWorkspace()).insights().getInsights();
-
-        expect(result).toMatchSnapshot();
+        const sanitizeResult = await sanitizeKeyWithNewValue(result, "uri", "sanitize_uri");
+        expect(sanitizeResult).toMatchSnapshot;
```

Comment:
`.toMatchSnapshot` should be probably function call? ( `.toMatchSnapshot()` )

Reply:
updated